src/app/components/jobs-slider/jobs-slider.component.scss

```diff
@@ -0,0 +1,109 @@
+@import '../../styles/colors';
+@import '../../styles/dimensions';
+@import '../../styles/typography';
+$job-card-border-radius: 8px;
+
+.home-jobs-slider-container {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 3em;
+  @media screen and (max-width: $md-device-width) {
+    padding: 3em 1em;
+  }
+
+  .home-jobs-slider-new-jobs-title {
+    color: $color-secondary-gray;
+  }
+
+  .home-jobs-slider-user-jobs-title {
+    color: $white;
+  }
+
+  .home-jobs-slider-show-all-jobs-link-container {
+    width: 100%;
+    display: flex;
+    flex-direction: column-reverse;
+
+    .home-jobs-slider-show-all-jobs-link {
+      cursor: pointer;
+      align-self: flex-end;
+      padding-left: 3em;
+    }
+
+    .home-jobs-slider-show-all-new-jobs-link {
+      color: $color-primary-cerise;
+    }
+
+    .home-jobs-slider-show-all-user-jobs-link {
+      color: $white;
+    }
+  }
+
+  .home-jobs-slider {
+    width: 100%;
+    margin-top: 15px;
+    overflow-y: scroll;
+    white-space: nowrap;
+
+    &::-webkit-scrollbar {
+      display: none;
+    }
+  }
+}
+
+.home-jobs-slider-new-jobs-container {
+  background: $white;
+}
+
+.home-jobs-slider-user-jobs-container {
+  background: $color-secondary-gray;
+}
+
+.home-jobs-applied-for-container {
+  display: flex;
+  flex-wrap: wrap;
+
+  .home-jobs-applied-for-header-container {
+    background: $color-gray-25;
+    border-bottom: 2px solid $color-primary-cerise;
+    width: 100%;
+
+    .home-jobs-applied-for-header {
+      margin-top: 15px;
+      margin-bottom: 15px;
+      margin-left: 48px;
+    }
+  }
+
+  .home-jobs-applied-for-job-item-container {
+    border-top: 1px solid $color-gray-25;
+    border-bottom: 1px solid $color-gray-25;
+    width: 100%;
+
+    &:hover {
+      opacity: 0.5;
+    }
+
+    .home-jobs-applied-for-job-item {
+      cursor: pointer;
+      display: flex;
+      width: 100%;
+      padding-left: 48px;
+
+      .home-jobs-applied-for-job-item-name {
+        margin-top: 15px;
+        margin-bottom: 15px;
+        @include truncate-owerflowing-text;
+      }
+
+      .home-jobs-applied-for-job-item-icon {
+        color: $color-primary-cerise;
```

Comment:
Properties should be ordered padding-left, margin-top, margin-right, margin-bottom, margin-left, color

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)